data-reconciliation-app/test/e2e-test/src/index.ts

```diff
@@ -0,0 +1,132 @@
+import Api from './api';
+import https from 'https';
+import fs from 'fs';
+import { member0DataPart1, member0DataPart2, member1Data, member2Data } from './data';
+import Host from './host';
+
+const serverUrl = 'https://127.0.0.1:8000';
+const certificateStorePath = './workspace/sandbox_common';
```

Comment:
If we make this generic to get server and certs_folder based on running mode, we can make this run in all of them.
is it our intention to run in all modes or just sandbox?

Reply:
In this iteration, that wasn't the intention. For now, I have just provided e2e scripts on a sandbox environment so we can run in CI/CD.

Reply:
Just IMO - I think this PR needs to fully implement all 3 hosting models. We have working bash scripts that test the application in virtual/docker/mccf. If we are introducing a new mechanism to test (TypeScript) then this needs to cover those same bases. Otherwise we have made it worse - I'm more than willing to help you if you need - but this PR at the moment is not complete. It's not far off finished though! :-)